src/openfl/display/DisplayObject.hx

```diff
@@ -893,7 +892,7 @@ class DisplayObject extends EventDispatcher implements IBitmapDrawable {
 			
 		}
 		
-		var renderParent = __renderParent != null ? __renderParent : parent;
+		var renderParent:DisplayObject = parent;
```

Comment:
Remove type definition here?

Reply:
No, because `parent` is `DisplayObjectContainer`, but we reassign it to `__maskTarget` later, which is just `DisplayObject`.